src/Data/Food/Retail.elm

```diff
@@ -0,0 +1,193 @@
+module Data.Food.Retail exposing (Conservation, all, ambient, computeImpacts, decode, displayNeeds, encode, fromString, toDisplay, toString)
```

Comment:
Nit: please expose these vertically as with all the other modules (elm-format will take care of it)

Reply:
done